src/components/custom/_signup-new.tsx

```diff
@@ -344,9 +249,12 @@ const SignupNew = ({
                     id="dm-signup-facebook"
                     type="button"
                     social
+                    bgColor="var(--color-blue-15)"
```

Comment:
We usually using snake_case for props, let's rename it a little bit
```suggestion
                    bg_color="var(--color-blue-15)"
```